### ISSUE TITLE
Setup fails with KeyError: 'WERKZEUG_SERVER_FD'

### ISSUE THREAD

**Issue by Anirudh58** [adamrehn/ue4-docker#233]
Output of the `ue4-docker info` command:

```
anip@blade:~/codes$ ue4-docker info
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.9) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
ue4-docker version:         0.0.96 (latest available version is 0.0.96)
Operating system:           Linux (Ubuntu 20.04.3 LTS, 5.13.0-37-generic)
Docker daemon version:      20.10.14
NVIDIA Docker supported:    Yes
Maximum image size:         No limit detected
Available disk space:       150.23 GiB
Total system memory:        15.53 GiB physical, 2 GiB virtual
CPU:                        6 physical, 12 logical (x86_64)
```

Additional details:

I am trying to run setup with `sudo ue4-docker setup`.

I get the error:

```
anip@blade:~/codes$ sudo ue4-docker setup
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.9) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/dist-packages/ue4docker/infrastructure/CredentialEndpoint.py", line 84, in _endpoint
    server.run(host="0.0.0.0", port=9876)
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 920, in run
    run_simple(t.cast(str, host), port, self, **options)
  File "/usr/local/lib/python3.8/dist-packages/werkzeug/serving.py", line 1047, in run_simple
    fd = int(os.environ["WERKZEUG_SERVER_FD"])
  File "/usr/lib/python3.8/os.py", line 675, in __getitem__
    raise KeyError(key) from None
KeyError: 'WERKZEUG_SERVER_FD'
Traceback (most recent call last):
  File "/usr/local/bin/ue4-docker", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/ue4docker/main.py", line 112, in main
    COMMANDS[command]["function"]()
  File "/usr/local/lib/python3.8/dist-packages/ue4docker/setup_cmd.py", line 156, in setup
    _setupLinux()
  File "/usr/local/lib/python3.8/dist-packages/ue4docker/setup_cmd.py", line 19, in _setupLinux
    endpoint.start()
  File "/usr/local/lib/python3.8/dist-packages/ue4docker/infrastructure/CredentialEndpoint.py", line 53, in start
    raise RuntimeError("failed to start the credential endpoint")
RuntimeError: failed to start the credential endpoint

```

- Are you accessing the network through a proxy server? **Yes/No** No

(The rest of the issue description goes here. If you're reporting a problem with building container images, be sure to include the full output of the `ue4-docker build` command, including the initial output lines that display the command line parameters used to invoke the build. If you're making a feature request, you can remove the template contents entirely, since the `ue4-docker info` output and related information is only needed for helping diagnose and reproduce bugs.)

**Comment by Polterageist** [comment]
Hi! It is a bug in dependency. I found workaround. Just install werkzeug package with forced 2.0.0 version.
I'm using pipenv so for me it just:
`pipenv install werkzeug==2.0.0`

**Comment by slonopotamus** [comment]
Actually, using whole Flask for a single HTTP endpoint sounds like _a bit of_ an overkill.

**Comment by davidism** [comment]
The issue is due to this line in your code: https://github.com/adamrehn/ue4-docker/blob/47c81a4480fbd06b5cda425ff214bcb628c0083b/ue4docker/infrastructure/CredentialEndpoint.py#L70

That is an internal variable, not an intended way to disable the server startup output. In fact, we absolutely don't want to disable that output, since it's very important that users understand the ramifications of running a development server in non-development situations. If you want to do so anyway, you can use `make_server` instead of `run_simple` to set up the server manually without the other behavior of `run_simple`.

**Comment by slonopotamus** [comment]
See #240

**Comment by slonopotamus** [comment]
Fixed in [0.0.97](https://github.com/adamrehn/ue4-docker/releases/tag/v0.0.97)
